staging/src/k8s.io/client-go/rest/config_test.go

```diff
@@ -139,13 +139,17 @@ func TestDefaultKubernetesUserAgent(t *testing.T) {
 }
 
 func TestRESTClientRequires(t *testing.T) {
-	if _, err := RESTClientFor(&Config{Host: "127.0.0.1", ContentConfig: ContentConfig{NegotiatedSerializer: api.Codecs}}); err == nil {
+	getGV := func() *schema.GroupVersion {
```

Comment:
add comment: "this is here since RESTClient might mutate the value we pass in"

(Is this true? It seems weird, why does RESTClient do this?)